### ISSUE TITLE
[Impeller] Add docs page for Impeller preview

### ISSUE THREAD

**Issue by loic-sharma** [flutter/website#8184]

**Comment by sfshaza2** [comment]
So, after more thought about this, I'm going to wait. Our general policy is that we document experimental features on the Flutter wiki. Feature that have hit stable are documented on the website. Perhaps we need to improve the wiki docs?

**Comment by chinmaygarde** [comment]
[I've already updated the wiki](https://github.com/flutter/flutter/wiki/Impeller-Scene) and linked it in the sidebar.

**Comment by loic-sharma** [comment]
> Our general policy is that we document experimental features on the Flutter wiki. 

@sfshaza2 From my understanding, we'd like to enable Impeller by default in the next release unless we find major issues. Thus, we want as many people as possible to try the Impeller preview to catch issues. Flutter's docs are much more visible than Flutter's wiki, so breaking that general policy could greatly help Impeller's launch.

**Comment by sfshaza2** [comment]
Wow, @loic-sharma, that's big news! Would it be for iOS only? Or would it work on other platforms, too? If we do enable Impeller by default, I agree that a page on the website is in order.  cc @timsneath

**Comment by timsneath** [comment]
Yeah, this is already available in the stable release but just off by default. 

The plan is to have it on by default in our next release on iOS, and available as a preview for Android in that timeframe. 

I'd be supportive of including this in the stable docs now on that basis.

**Comment by chinmaygarde** [comment]
Just following up on issues linked from the dashboard. Is this still in progress?

**Comment by sfshaza2** [comment]
It's back on the plan, but I haven't created the page yet.
